core/dbt/parser/manifest.py

```diff
@@ -333,15 +332,214 @@ def get_full_manifest(
 
         return manifest
 
+    def load_macros_and_tests(
+        self,
+        project_parser_files: Dict[str, Dict[str, List[str]]],
+        orig_project_parser_files: Dict[str, Dict[str, List[str]]],
+    ) -> None:
+        """
+        Load Macros and tests
```

Comment:
Can we add a bit more in the description about what objects will be changed after running this function?